.ruff.toml

```
@@ -0,0 +1,39 @@
+target-version = "py39"
+line-length = 110
+exclude = [
+  ".git,",
+  "__pycache__",
+  "build",
+  "CHIMERApy/version.py",
+]
+
+[lint]
+select = ["E", "F", "W", "UP", "PT"]
+extend-ignore = [
+  # pycodestyle (E, W)
+  "E501", # LineTooLong                             # TODO! fix
+  # pytest (PT)
+  "PT001", # Always use pytest.fixture()
+  "PT004", # Fixtures which don't return anything should have leading _
+  "PT007", # Parametrize should be lists of tuples  # TODO! fix
+  "PT011", # Too broad exception assert             # TODO! fix
+  "PT023", # Always use () on pytest decorators
```

Maybe this just remove the one that were marked to be fixed?
```suggestion
  # pytest (PT)
  "PT001", # Always use pytest.fixture()
  "PT004", # Fixtures which don't return anything should have leading _
  "PT023", # Always use () on pytest decorators
```